template_package/etc/bash_completion.d/mdtool_completion.sh

```diff
@@ -31,9 +31,6 @@ _mdtool_complete()
 			"register")
 			COMPREPLY=( $(compgen -W "read write" -- $cur) )
 			;; 
-			"setup")
```

Comment:
why delete while case? should only delete homing i think

Reply:
True, fixed